### PR TITLE
[STA Travel.com] New ruleset

### PR DESCRIPTION
**Pull request by youdly**

### CODE REVIEW

File: src/chrome/content/rules/STA-Travel.com.xml [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+	<target host="api.partners.my.statravel.com" />
+	<target host="rail.statravel.com" />
+	<target host="tours.statravel.com" />
+		<test url="http://tours.statravel.com/content/images/icon-cal-small.png" />
+	<target host="usvpn.statravel.com" />
+
+	<securecookie host=".+" name=".+" />
+
+	<!-- http redirect -->
+	<exclusion pattern="^http://www\.statravel\.com/(blog|tours)" />
```
Comment (J0WI) [inline src/chrome/content/rules/STA-Travel.com.xml]: Please add a trailing slash for blog

File: src/chrome/content/rules/STA-Travel.com.xml [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+		<test url="http://www.statravel.com/blogs/" />
```
Comment (J0WI) [inline src/chrome/content/rules/STA-Travel.com.xml]: Should be `blog`, not `blogs`

File: src/chrome/content/rules/STA-Travel.com.xml [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+		<test url="http://www.statravel.com/tours-worldwide.htm" />
```
Comment (J0WI) [inline src/chrome/content/rules/STA-Travel.com.xml]: Are there any other `tours*` or could you use a more restrictive regex?

Comment (youdly) [reply src/chrome/content/rules/STA-Travel.com.xml]: `/tours.htm` is also valid, redirects to `/tours-worldwide.htm`.

**Pull request merged**